src/ToolBox/SOS/Strike/strike.h

```diff
@@ -76,6 +76,21 @@
 #ifndef PAL_STDCPP_COMPAT
 #include <malloc.h>
 #endif
+
+#if defined(_MSC_VER)
+#define alloca _alloca
```

Comment:
I think you can simplify this whole thing to (in pal.h too) to:

```
#ifdef FEATURE_PAL
#ifndef alloca
#define alloca  __builtin_alloca
#endif
#ifndef _alloca
#define _alloca alloca
#endif
#endif // FEATURE_PAL
```

The #ifndef checks should fix the build breaks on FreeBSD and CentOS.